backend/app/controllers/enrollments_controller.rb

```diff
@@ -53,10 +53,10 @@ def index
       # silently fail, if the filter is not formatted properly we do not apply it
     end
 
-    page = params.fetch(:page, "0")
-    size = params.fetch(:size, "10")
-    size = "100" if size.to_i > 100
-    @enrollments = @enrollments.page(page.to_i + 1).per(size.to_i)
+    page = params[:page] || 0
```

Comment:
I am curious : is there a reason to prefer direct access of the variable instead of fetch ?

Reply:
After some recherche and testing the 2 ways, it seems that we are gaining in performance to use direct access of the variable instead of using fetch. 

The advantage of using .fetch is to provide a second parameter which acts as a default value ex : `fetch(:size "10")` 

We can also assign default value  `params[:per_page]` with kaminari gems (25 if I'm not wrong) and force to use only 10 for this table using `params[:per_page] || 10`